src/github_review_prompts/utils/rate_limit_handler.py

```
@@ -0,0 +1,435 @@
+    def _execute_with_rate_limit(self, func: Callable, api_type: str, *args, **kwargs):
+        """レート制限を考慮した関数実行"""
+        max_retries = 3
+        retry_count = 0
+
+        while retry_count < max_retries:
+            try:
+                # 事前チェック：レート制限に近づいている場合は待機
+                self._pre_request_check(api_type)
+
+                # 関数実行
+                start_time = time.time()
```

Ruff (0.12.2)

Local variable `start_time` is assigned to but never used

Remove assignment to unused variable `start_time`

(F841)